src/main/java/org/medicmobile/webapp/mobile/OpenSettingsDialogFragment.java

```diff
@@ -13,15 +13,16 @@
 
 import java.time.Clock;
 
-@SuppressLint("ValidFragment")
 public class OpenSettingsDialogFragment extends Fragment {
 
-	private final View view;
 	private int fingerTapCount = 0;
 	private long lastTimeTap = 0;
 	private GestureHandler swipeGesture;
 	private static final int TIME_BETWEEN_TAPS = 500;
 
+	public OpenSettingsDialogFragment() {
```

Comment:
I think this is unnecessary as the default constructor will work, right?

Reply:
Yes, it does work without this. Removing this.